rskj-core/src/main/java/co/rsk/validators/ValidTxExecutionListsEdgesRule.java

```diff
@@ -19,29 +19,31 @@ public class ValidTxExecutionListsEdgesRule implements BlockValidationRule {
     @Override
     public boolean isValid(Block block) {
         short[] edges = block.getHeader().getTxExecutionListsEdges();
-        int nTxs = block.getTransactionsList().size();
 
-        if (edges == null) {
+        if (edges == null || edges.length == 0) {
```

Comment:
Why remove the second check?

Reply:
What did I remove? 🤔 If length is zero why would we iterate and all that.

`Constants.getMaxTransactionExecutionThreads()` must be >1